src/Application/Actions/Donations/Create.php

```diff
@@ -147,8 +150,20 @@ protected function action(Request $request, Response $response, array $args): Re
                 $donation->setCampaign($campaign);
             }
 
+            if ($forNewPaymentElement) {
+                // We don't need to set specific payment methods here, they are set on the client side by the payment
```

Comment:
@bdsl This doesn't feel like the best approach to me – what if somebody uses modified frontend code to try out methods we don't expect against our backend? I think we should support just the methods we expect.

Reply:
Cool – presumably we could still do the reverse and allow auto methods everywhere (along with making sure those we don't want are off in the account) @bdsl?

Reply:
See https://github.com/thebiggive/matchbot/pull/588